inference-engine/src/transformations/src/itt.hpp

```diff
@@ -11,6 +11,7 @@
 
 #include <openvino/cc/selective_build.h>
 #include <openvino/itt.hpp>
+#include <ngraph/pass/itt.hpp>
```

Comment:
Need to replace code in snippets then

Reply:
We didn't have CC macros in snippets. I added these macros to template plugin.

Reply:
Really?
https://github.com/openvinotoolkit/openvino/blob/315c8d4eec75d1efbce553434ed5db5df6d97665/inference-engine/src/snippets/src/itt.hpp#L33-L59

Reply:
Good catch